src/TMedia.cpp

```diff
@@ -607,7 +607,7 @@ TMediaPlayer TMedia::getMediaPlayer(TMediaData& mediaData)
     return pPlayer;
 }
 
-TMediaPlayer TMedia::matchMediaPlayer(TMediaData& mediaData, QString absolutePathFileName)
+TMediaPlayer TMedia::matchMediaPlayer(TMediaData& mediaData, const QString& absolutePathFileName)
```

Comment:
These conversions from a `QString` pass by value to a constant pass-by-reference seem okay to me.